src/test/kotlin/no/nav/syfo/cronjob/dialogmotesvar/PublishDialogmotesvarServiceSpek.kt

```diff
@@ -95,6 +95,31 @@ class PublishDialogmotesvarServiceSpek : Spek({
                 dialogmotesvarList.size shouldBeEqualTo 1
             }
 
+            it("Behandlers answer is not valid and will not be published") {
+                val identifiers = database.connection.createNewDialogmoteWithReferences(
+                    newDialogmote = generateNewDialogmoteWithBehandler(
+                        UserConstants.ARBEIDSTAKER_FNR,
+                        DialogmoteStatus.INNKALT,
+                    )
+                )
+                val varselId = database.connection.createBehandlerVarsel(
+                    UUID.randomUUID(),
+                    MotedeltakerVarselType.INNKALT,
+                    identifiers.motedeltakerBehandlerIdPair?.first,
+                )
+                val kommerIkkeSvarUuid = UUID.randomUUID()
+                database.connection.createBehandlerVarselSvar(
+                    svarUuid = kommerIkkeSvarUuid,
+                    varselId = varselId,
+                    svarType = DialogmoteSvarType.KOMMER_IKKE,
+                    valid = false,
+                )
```

Comment:
Vet ikke om jeg misforstår, men settes den noen gang til `false` i selve servicen/business-logikken? Eller er denne PRen kun for å fjerne de fra hva som publiseres på kafka-topicen?

Reply:
Jeg holder på fikse logikk for å sette valid til false, men det kommer som en egen PR, ja👍 Bra du spør, så vi ikke legger til ting unødvendig! 🏆